src/go/k8s/main.go

```diff
@@ -284,9 +284,10 @@ func main() {
 	}()
 
 	if err = (&redpandacontrollers.RedpandaReconciler{
-		Client:        mgr.GetClient(),
-		Scheme:        mgr.GetScheme(),
-		EventRecorder: mgr.GetEventRecorderFor("RedpandaReconciler"),
+		Client:          mgr.GetClient(),
+		Scheme:          mgr.GetScheme(),
+		EventRecorder:   mgr.GetEventRecorderFor("RedpandaReconciler"),
+		RequeueHelmDeps: 10 * time.Second,
```

Comment:
Can this be configurable?

Reply:
This is definitely going to be. Flags are something to be added later as we figure out what we need. I'll ask a todo do i don't forget.